openmetadata-ui/src/main/resources/ui/src/pages/tags/Form.tsx

```diff
@@ -124,6 +136,9 @@ Form.propTypes = {
     description: PropTypes.string.isRequired,
     categoryType: PropTypes.string.isRequired,
   }).isRequired,
+  errorData: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+  }).isRequired,
```

Comment:
why we are making it as required?
In form prop it is marked as optional 
```  errorData?: { [key: string]: string | undefined };```

Reply:
Nice observation, Thanks :)